auth/src/main/res/layout/fui_check_email_layout.xml

```diff
@@ -1,33 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
+<ScrollView
```

Comment:
I think It would be good to remove ScrollView and change LinearLayout with ConstraintLayout as you've already done in layouts below.

I'm pretty sure that adding `ScrollView` like adding any other layer of UI  to xml file would finally decrease app performance. Although in that situation, where layout is a relatively small, it would be pretty hard to notice. 

As @samtstern said, there's no reason to use `ScrollView` there, as it won't more space on small device.

Reply:
@samtstern In an extreme case where a user has a small screen and is in multi-window mode, they could get to the point where they need to scroll:
<img src="https://user-images.githubusercontent.com/9490724/33583996-0ade1816-d912-11e7-8f8b-da180813758c.png" width="50%" height="50%" />

That was on my Pixel 2 XL which has a pretty big aspect ratio so I can image the need to scroll on a normal screen. Then again, this is a pretty rare edge case.

@samtstern @piotrek1543 As for performance concerns, I've Googled around and didn't find anything. I think since it's just one top-level layout, performance is basically irrelevant. (The activity will take _way_ longer to start than the extra onMeasure calls from using a scrollview.)

But as I said, I'm not going to be nit picky here so it's your call @samtstern. 😄

Reply:
LGTM thanks for justifying.